### ISSUE TITLE
Installed image not booting, only booting to berrboot installer

### ISSUE THREAD

**Issue by jdmoore0883** [maxnet/berryboot#367]
Taking my queues from [http://www.berryterminal.com/doku.php/berryboot](http://www.berryterminal.com/doku.php/berryboot) , I am booting BerryBoot, and installing an image to a USB Stick. I get a successful install, and a prompt to reboot. I reboot, and back to the installer again.
This persists across all images, and multiple USB devices.

Ultimately, I want to be able to multi-boot multiple USB drives (1 USB for Raspbian, another for RetroPi, another for something else, etc...), and I am happy to admit that I am doing something wrong, or missing something... But I have gone over the details multiple times, and can't seem to identify what, if anything, I am doing wrong wrong or missing.

Any ideas?

**Comment by maxnet** [comment]
Tried a different SD card?

**Comment by jdmoore0883** [comment]
Yes, I have also tried another card, but of the same brand I believe, maybe even same size and model. I will test a few others if I can tomorrow.

**Comment by jdmoore0883** [comment]
Multiple Brands and sizes of SD Cards tested, along with some more USB sticks, all with the same results. I have also tried installing the Pi Image to the SD Card as well. No matter what I do, I get success messages, but upon reboot I am right back to the installer; at no time do I see a boot menu editor.

**Comment by maxnet** [comment]
Make sure to properly format the SD cards, put the files on it, and use the "safe remove hardware" function in the OS you are using to copy the files.

Have seen this behavior before with SD cards with corrupted FAT file system.
Pi firmware was using a different FAT when reading cmdline.txt, than Berryboot.

**Comment by jdmoore0883** [comment]
Though I typically do so to begin with, I have made a few more attempts, being sure to do as you suggest. The issue persists regardless.

**Comment by maxnet** [comment]
cmdline.txt does not list a datadev after installation?

**Comment by jdmoore0883** [comment]
Oh wow do I ever feel dumb now... I got it going, and here's how/why.
To start, I wasn't entirely truthful about the instructions I was following... I had followed the Headless VNC install as per:
[http://www.berryterminal.com/doku.php/berryboot/headless_installation](http://www.berryterminal.com/doku.php/berryboot/headless_installation)
Therein, is a key detail I had missed, and only noted after much, fine review:
"...append to cmdline.txt **on the same line** as the existing options:..."
I thought I would be smart, and try to keep the file nice and clean by adding the VNC options to another line, and when it didn't work on the second line, I put it first instead, and THAT worked, but I encountered the issue as described here. As soon as I properly **FOLLOWED THE INSTRUCTIONS**, BOOM!, it all just worked.
While I am sorry to have wasted your time maxnet, it is a good exercise in breaking things and detailing how it breaks, so it's not a TOTAL waste of time...
